Automatic Validation ended with: 
> ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [2024.1, 2024.1]]

This situation usually requires that the existing manifest be removed before the new one can be added.

--- a/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.installer.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.installer.yaml
@@ -1,0 +1,23 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.2.13
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSuccessCodes:
+- 1223
+UpgradeBehavior: install
+ReleaseDate: 2024-09-17
+AppsAndFeaturesEntries:
+- DisplayVersion: "2024.1"
+ElevationRequirement: elevatesSelf
+Installers:
+- Architecture: x64
+  InstallerUrl: https://redirector.gvt1.com/edgedl/android/studio/install/2024.1.2.13/android-studio-2024.1.2.13-windows.exe
+  InstallerSha256: 71D018D5685073E6D59E6B91358E0240757C06241B0A1C94E532801420965F6D
+ManifestType: installer
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.locale.en-US.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.locale.en-US.yaml
@@ -1,0 +1,44 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.2.13
+PackageLocale: en-US
+Publisher: Google LLC
+PublisherUrl: https://developer.android.com/
+PublisherSupportUrl: https://developer.android.com/studio/intro
+PrivacyUrl: https://policies.google.com/privacy
+PackageName: Android Studio
+PackageUrl: https://developer.android.com/studio
+License: Android Software Development Kit License Agreement
+LicenseUrl: https://developer.android.com/studio/terms
+Copyright: Copyright © 2000–2024 Google
+CopyrightUrl: https://developer.android.com/studio/terms
+ShortDescription: The official Integrated Development Environment (IDE) for Android app development.
+Description: |-
+  Android Studio is the official Integrated Development Environment (IDE) for Android app development.
+  Based on the powerful code editor and developer tools from IntelliJ IDEA,
+  Android Studio offers even more features that enhance your productivity when building Android apps.
+Moniker: android-studio
+Tags:
+- aab
+- adb
+- android
+- androidstudio
+- apk
+- app
+- debug
+- development
+- emulator
+- fastboot
+- ide
+- intellij
+- studio
+ReleaseNotesUrl: https://developer.android.com/studio/releases
+Documentations:
+- DocumentLabel: Troubleshoot
+  DocumentUrl: https://developer.android.com/studio/troubleshoot
+- DocumentLabel: Known Issues
+  DocumentUrl: https://developer.android.com/studio/known-issues
+ManifestType: defaultLocale
+ManifestVersion: 1.6.0

--- a/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.yaml
+++ b/manifests/g/Google/AndroidStudio/2024.1.2.13/Google.AndroidStudio.yaml
@@ -1,0 +1,8 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Google.AndroidStudio
+PackageVersion: 2024.1.2.13
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.6.0

